[layout] north arrow in toolbox (#4272)
qgis/QGIS-Documentation: source/docs/user_manual/print_composer/composer_items/composer_image.rst

.. figure:: img/image_mainproperties.png
   :align: center

>    

Author: ```suggestion
```

You can also apply a declination :guilabel:`Offset` to the picture rotation.

North Arrows
------------

> You can use the |northArrow| :sup:`Add North Arrow`  shortcut directly from the Toolbox in the print layout and you can pimp it as a :guilabel:`Picture` item as described above (see also figure_layout_image_).

Reviewer: ```suggestion
You can use the |northArrow| :sup:`Add North Arrow` shortcut directly from
the Layout Toolbox. It has a lot in common with the :guilabel:`Picture` item
described above (see also figure_layout_image_).
```

Reviewer 2: I'm not convinced the suggestion is better. The "Add north Arrow" IS an "Add picture" button except that it defaults to north arrows categories. It does not have a lot in common with picture (_looks unclear and ambiguous to me_); it's a picture and shares all of its properties.

Reviewer: I first reacted on the use of "pimp", and then I went on to suggest some more changes. Also, the number of characters per line is far over our "limit".  It would be nice if someone could come up with a better alternative.
